### PR TITLE
Changed the STD Awareness logo in the Learn screen

### PR DESCRIPTION
**Pull request by PantherDave**

### CODE REVIEW

**Review by edumorlom — CHANGES_REQUESTED** [review]
Please fix conflicts and remove previous icon from the repository completely.

File: src/Components/Learn.jsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { TouchableHighlight, Text, View, ScrollView } from "react-native";
 import appStyles from "./AppStyles";
 import SelectionButton from "./SelectionButton";
 import butterfly from "../../assets/butterfly.png";
```
[the remark this answers is not in the document]
Comment (PantherDave) [reply src/Components/Learn.jsx:5]: Should i remove the .png file from assets as well?

[the remark this answers is not in the document]
Comment (edumorlom) [reply src/Components/Learn.jsx:5]: yes please

[the remark this answers is not in the document]
Comment (PantherDave) [reply src/Components/Learn.jsx]: butterfly.png is gone now

**Review by edumorlom — APPROVED** [review]
great thanks!

**Pull request merged**